onnxscript/irbuilder.py

```diff
@@ -370,13 +370,15 @@ def to_proto(f):
         for n in self.stmts:
             if n.callee.opset.domain not in opsets:
                 opsets[n.callee.opset.domain] = n.callee.opset.version
+
```

Comment:
I would add a comment about the significance of the ordering here.

Reply:
Hmmm ... actually, I didn't check this carefully ... changed it to WIP ... need to check a couple of things

Reply:
Ok, fixed it, hopefully it is okay now.